modules/electrophysiology_browser/php/models/ElectrophysioFile.class.inc

```diff
@@ -0,0 +1,86 @@
+<?php declare(strict_types=1);
+/**
+ * Contains the ElectrophysioFile class used for electrophysiological browser
+ *
+ * PHP Version 7
+ *
+ * @category Electrophysiology
+ * @package  Electrophysiology
+ * @author   Loris Team <loris.mni@bic.mni.mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+namespace LORIS\electrophysiology_browser\Models;
+
+/**
+ * This object represents an electrophysiology file in the physiological_file table.
+ *
+ * @category Electrophysiology
+ * @package  Electrophysiology
+ * @author   Loris Team <loris.mni@bic.mni.mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class ElectrophysioFile implements \LORIS\Data\DataInstance
+{
+    private $_fileData   = array();
+    private $_parameters = array();
+
+    /**
+     * Construct a BIDSFile
+     *
+     * @param integer $physiologicalFileID The PhysiologicalFileID to be loaded
+     */
+    function __construct(int $physiologicalFileID)
+    {
+
+        $db = \NDB_Factory::singleton()->database();
+
+        $query    = "SELECT * 
```

Comment:
This should be an explicit list of files, not a `SELECT *`

Reply:
ok. Done